slave-ansible/Dockerfile

```diff
@@ -1,7 +1,9 @@
-FROM openshift/jenkins-slave-base-centos7
+FROM docker.io/fhwendy/jenkins-slave-base-centos7:RHMAP15237
 
 MAINTAINER Oleg Matskiv <omatskiv@redhat.com>
```

Comment:
We are doing changes to the source of the image source that originally was openshift/jenkins-slave-base-centos7, to allow for usage with jenkins-swarm plugin. So not really a layer, more of a fork?